ui/src/pages/ShoeMap.tsx

```diff
@@ -1,11 +1,37 @@
 import * as React from "react";
+import styled from "styled-components";
+import { Map, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+
 import { TitleText } from "../styles/typography";
 
+const StyledMap = styled(Map)`
+  height: 70vh;
+  width: 90vw;
+  display: flex;
+`;
+
+const MapContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
 export const ShoeMap: React.FC = () => {
+  const zoom = 12;
+  const currentLocation = { lat: 43.4723, lng: -80.5449 };
+
   return (
-    <div>
-      <TitleText>ShoeMap</TitleText>
-      <span> test text</span>
-    </div>
+    <MapContainer>
+      <TitleText>Shoe Map</TitleText>
+      <StyledMap center={currentLocation} zoom={zoom}>
+        <TileLayer
+          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
```

Comment:
I think we discussed using mapbox instead of openstreetmap, since even though mapbox is not open source it will be free for our usage level.

Reply:
Ooh okay I'll change it